Docs build failure

```
#24 [build-docs 4/4] RUN pnpm build
#24 37.31 ✔ Client: Compiled successfully in 31.76s
#24 40.72 ✔ Server: Compiled successfully in 35.15s
#24 56.43 [ERROR] Unable to build website for locale en.
#24 56.43 [ERROR] Error: Docusaurus found broken links!
#24 56.43 
#24 56.43 Please check the pages of your site in the list below, and make sure you don't reference any path that does not exist.
#24 56.43 Note: it's possible to ignore broken links with the 'onBrokenLinks' Docusaurus configuration, and let the build pass.
#24 56.43 
#24 56.43 Exhaustive list of all broken links found:
#24 56.43 
#24 56.43 - On source page path = /docs/getting-started/vscode-ssh:
#24 56.43    -> linking to ../../../../../config/ec2-template.yml (resolved as: /config/ec2-template.yml)
#24 56.43 
#24 56.46  ELIFECYCLE  Command failed with exit code 1.
#24 ERROR: process "/bin/sh -c pnpm build" did not complete successfully: exit code: 1

#23 [build-app 4/4] RUN pnpm build
#23 CANCELED
------
 > [build-docs 4/4] RUN pnpm build:
#24 56.43 
#24 56.43 Please check the pages of your site in the list below, and make sure you don't reference any path that does not exist.
#24 56.43 Note: it's possible to ignore broken links with the 'onBrokenLinks' Docusaurus configuration, and let the build pass.
#24 56.43 
#24 56.43 Exhaustive list of all broken links found:
#24 56.43 
#24 56.43 - On source page path = /docs/getting-started/vscode-ssh:
#24 56.43    -> linking to ../../../../../config/ec2-template.yml (resolved as: /config/ec2-template.yml)
#24 56.43 
#24 56.46  ELIFECYCLE  Command failed with exit code 1.
------
Dockerfile:97
--------------------
  95 |     COPY --from=dependencies-docs /docs/node_modules ./node_modules
  96 |     
  97 | >>> RUN pnpm build
  98 |     
  99 |     ## Deploy ######################################################################
--------------------
error: failed to solve: process "/bin/sh -c pnpm build" did not complete successfully: exit code: 1
Error: buildx failed with: error: failed to solve: process "/bin/sh -c pnpm build" did not complete successfully: exit code: 1
```

--- a/src/web/docusaurus/docs/getting-started/vscode-ssh.md
+++ b/src/web/docusaurus/docs/getting-started/vscode-ssh.md
@@ -2,7 +2,7 @@
 
 :::info
 
-This guide has also been translated into a [CloudFormation](https://aws.amazon.com/cloudformation/) Template that can be used to automatically create the same resources. See [config/e2c-template.yml](../../../../../config/ec2-template.yml).
+This guide has also been translated into a [CloudFormation](https://aws.amazon.com/cloudformation/) Template that can be used to automatically create the same resources. See `config/e2c-template.yml`.
 
 :::
 

